instance/models/openedx_instance.py

```diff
@@ -67,58 +71,77 @@ def __str__(self):
         return "{} ({})".format(self.name, self.domain)
 
     @property
-    def domain(self):
+    def internal_lms_domain(self):
         """
-        Instance domain name
+        Instance internal LMS domain name (eg. 'master.sandbox.opencraft.com').
         """
         return '{0.sub_domain}.{0.base_domain}'.format(self)
 
     @property
-    def url(self):
+    def internal_lms_preview_domain(self):
         """
-        LMS URL
+        Instance internal LMS preview domain name (eg. 'preview-master.sandbox.opencraft.com').
         """
-        return u'{0.protocol}://{0.domain}/'.format(self)
+        return 'preview-{0.sub_domain}.{0.base_domain}'.format(self)
 
     @property
-    def studio_sub_domain(self):
+    def internal_studio_domain(self):
         """
-        Studio sub-domain name (eg. 'studio-master')
+        Instance internal Studio domain name (eg. 'studio-master.sandbox.opencraft.com').
         """
-        return 'studio-{}'.format(self.sub_domain)
+        return 'studio-{0.sub_domain}.{0.base_domain}'.format(self)
 
     @property
-    def studio_domain(self):
+    def domain(self):
         """
-        Studio full domain name (eg. 'studio-master.sandbox.opencraft.com')
+        LMS domain name.
+        Returns external domain if present, otherwise returns internal domain.
```

Comment:
@itsjeyd Thanks, I added a blank line. I also some info about external vs internal domain, but I did that in a comment above external domain declarations rather than in the docstring of this method.